src/modeler/AutoRest.Swagger/Validation/Core/RulesExtensions.cs

```diff
@@ -64,6 +64,6 @@ public static IEnumerable<Rule> GetValidationRules(this Type type)
         /// <summary>
         /// The collection of rules that apply to all properties regardless of other rules.
         /// </summary>
-        public static IEnumerable<Rule> UniversalRules = new[] { new NoControlCharacters() };
+        public static IEnumerable<Rule> UniversalRules = new List<Rule>();
```

Comment:
the core should be doing no such thing (and doesn't seem to in the test I just did), can I have the test case? let's make sure we're not missing a bug.